data/mods/DinoMod/monsters/zed-dinosaur.json

```
@@ -1007,19 +1033,20 @@
     "armor_bash": 8,
     "armor_cut": 8,
     "armor_bullet": 3,
+    "burn_into": "mon_zrachiosaurus_scorched",
     "fungalize_into": "mon_zrachiosaurus_fungus",
     "upgrades": { "half_life": 14, "into": "mon_zrachiosaurus" }
   },
   {
     "type": "MONSTER",
     "id": "mon_zalamosaurus_juvenile",
-    "name": { "str": "juvenile Alamosaurus zombie" },
+    "name": { "str": "juvenile sauropod zombie" },
     "copy-from": "mon_zapatosaurus_juvenile",
     "looks_like": "mon_apatosaurus",
     "armor_bash": 7,
     "armor_cut": 9,
-    "armor_bullet": 1,
     "description": "The corpse of a small, four-legged, herbivorous dinosaur juvenile, with natural bone armor plates.",
```

```suggestion
    "description": "The corpse of a small, four-legged, herbivorous dinosaur juvenile, with cracked bone armor plates.",
```
To explain the loss of armor_bullet.